browser/src/control/Control.UIManager.js

```diff
@@ -644,7 +644,7 @@ L.Control.UIManager = L.Control.extend({
 		var obj = $('.unfold');
 		obj.removeClass('w2ui-icon unfold');
 		obj.addClass('w2ui-icon fold');
-
+		$('#tb_editbar_item_fold').prop('title', 'Hide Menu');
```

Comment:
we need to encapsulate the string again here. Instead of `'Hide Menu'` please do `_('Hide Menu')`. We use this formatting to mark strings that should be translated